cmd/flags/es/options.go

```diff
@@ -109,7 +109,7 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 	flagSet.String(
 		nsConfig.namespace+suffixServers,
```

Comment:
maybe change suffixServers to suffixServerURLs? but the internal variable can be kept as servers.